### ISSUE TITLE
I got some trouble about train.py

### ISSUE THREAD

**Issue by door5719** [andy-yun/pytorch-0.4-yolov3#3]
I got some trouble about train.py......

pyhton 3.5 pytorch 0.4

How to use train.py?

Anybody ran train.py succesfully?

**Comment by andy-yun** [comment]
You can see the usage of train.py on youtube videos. 
Just you typed python train.py -c config_files -d data_files -w weight_files [-r] 
In my repository, you need the base weights for example yolov3.weights.
-r means that it resets the epoch number (actually model seen parameters) saved in the weight file.

**Comment by door5719** [comment]
is  this program runnig on win10 python 3.5 or ubantu?

**Comment by door5719** [comment]
Thank you very much~! this problem is sloved! Thank you!

**Comment by door5719** [comment]
@andy-yun ,sir,i want to know how to change the loss function,just as box , conf and class .I want to train tiny-yolo model with other datas(such as from yolov3) but not the ground truth,just like knowledge distillation learning style. The idear is from paper(Object detection at 200 Frames Per Second) i got from https://arxiv.org/pdf/1805.06361.pdf. can you give same advice? Thank you ~~

**Comment by andy-yun** [comment]
Tiny-yolo uses the region layer, thus, you should modified loss part of the region_layer.py.
Losses are calculated in forward path. You can easily find the loss_coord, loss_conf, loss_cls between 167 and 169 lines. In region layer, it uses ground truth information. Therefore, if you want to change the target as distillation information as you suggested, you will change the tcoord, tconf, tcls values returned from build_tagets(). Good luck.

**Comment by door5719** [comment]
I'm glad to get your reply.Thank you ,sir.

**Comment by door5719** [comment]
Sir ,I want to know the meaning of "max_batches".
In AlexeyAB/darknet' YOLOV3(c++),"max_batches" reading from cfg may mean max epochs.
in your project "max epochs" may not get from cfg(such as yolov3tiny.cfg,there is no option like max epochs).Is there some misunderstanding I got?

**Comment by door5719** [comment]
@andy-yun  ,sir, in this work,i try insert :print("pred_boxes[0] =",pred_boxes[0] ) after region_layer.py ,line 145, but there is no different with before display.what's the reason?

**Comment by andy-yun** [comment]
That means your configuration did not call region layer. You should check the configuration file.
The region_layer.py is used when the [region] is appeared on your cfg file.

**Comment by door5719** [comment]
Thank you sir
